matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/query/RoomMemberEntityQueries.kt

```diff
@@ -32,3 +33,12 @@ internal fun RoomMemberSummaryEntity.Companion.where(realm: Realm, roomId: Strin
     }
     return query
 }
+
+internal fun RoomMemberSummaryEntity.Companion.updateUserPresence(realm: Realm, userId: String, userPresenceEntity: UserPresenceEntity) {
+    realm.where<RoomMemberSummaryEntity>()
+            .equalTo(RoomMemberSummaryEntityFields.USER_ID, userId)
+            .findAll()
+            .map {
+                it.userPresence = userPresenceEntity
```

Comment:
is it necessary to do this? The link between ` it.userPresence` and `userPresenceEntity` is always the same, no?

Reply:
Well, I think you are partially right, we need that only for the first linking, after that it will auto update the changes, so I simply add in the query` .isNull(RoomMemberSummaryEntityFields.USER_PRESENCE.`$`)` so now it will only do that for the first linking. You think there is a better way  for the initial linking?